src/tools/generate/generate_sitemap.js

```diff
@@ -91,14 +91,14 @@ const set_min_date = (date) => {
   }
 };
 
-const strip_html_extension = (url) => {
+const convert_file_name = (url) => {
   if ( url.substr(url.length - 10) == "index.html" ) {
     return url.substr(0, url.length - 10);
   };
   if ( url.substr(url.length - 5) == ".html" ) {
-    return url.substr(0, url.length - 5);
+    return url.substr(0, url.length - 5).replace('_','-');
```

Comment:
Does this do a global replace? TOC page still has an underscore. Same for next `replace` call.

Reply:
Nope it doesn't. Stupid language. Fixed with a regex. Bout time we had [replaceAll](https://v8.dev/features/string-replaceall)!